front/app/containers/SiteMap/Project.tsx

```diff
@@ -15,52 +15,50 @@ import useProject from 'hooks/useProject';
 // intl
 import { FormattedMessage } from 'utils/cl-intl';
 import messages from './messages';
-import { IAdminPublicationContent } from 'hooks/useAdminPublications';
 
 interface Props {
-  adminPublication: IAdminPublicationContent;
+  projectId: string;
   hightestTitle: 'h3' | 'h4';
 }
 
-export default ({ adminPublication, hightestTitle }: Props) => {
-  const projectId = adminPublication.publicationId;
+const Project = ({ projectId, hightestTitle }: Props) => {
   const { events } = useEvents({
     projectIds: [projectId],
   });
   const project = useProject({ projectId });
 
-  const TitleComponent = hightestTitle === 'h3' ? H3 : H4;
+  const TitleComponent = { h3: H3, h4: H4 }[hightestTitle];
 
-  return (
-    <>
-      <TitleComponent>
-        <T value={adminPublication.attributes.publication_title_multiloc} />
-      </TitleComponent>
-      <ul>
-        <li>
-          <Link
-            to={`/projects/${adminPublication.attributes.publication_slug}/info`}
-          >
-            <FormattedMessage {...messages.projectInfo} />
-          </Link>
-        </li>
-        {!isNilOrError(project) && (
-          <>
-            {project.attributes.process_type === 'continuous' ? (
-              <ContinuousProject key={project.id} project={project} />
-            ) : (
-              <TimelineProject key={project.id} project={project} />
-            )}
-            {!isNilOrError(events) && events.length > 0 && (
-              <li>
-                <Link to={`/projects/${project.attributes.slug}/events`}>
-                  <FormattedMessage {...messages.projectEvents} />
-                </Link>
-              </li>
-            )}
-          </>
-        )}
-      </ul>
-    </>
-  );
+  if (!isNilOrError(project)) {
+    return (
+      <>
+        <TitleComponent>
+          <T value={project.attributes.title_multiloc} />
+        </TitleComponent>
+        <ul>
+          <li>
+            <Link to={`/projects/${project.attributes.slug}/info`}>
+              <FormattedMessage {...messages.projectInfo} />
+            </Link>
+          </li>
+          {project.attributes.process_type === 'continuous' ? (
+            <ContinuousProject key={project.id} project={project} />
+          ) : (
+            <TimelineProject key={project.id} project={project} />
+          )}
+          {!isNilOrError(events) && events.length > 0 && (
+            <li>
+              <Link to={`/projects/${project.attributes.slug}/events`}>
+                <FormattedMessage {...messages.projectEvents} />
+              </Link>
+            </li>
+          )}
+        </ul>
+      </>
+    );
+  }
+
+  return null;
 };
+
+export default Project;
```

Comment:
Export a named component because it's hard to find a component in the DOM without.